contextcheck.go

```diff
@@ -551,6 +559,10 @@ func (r *runner) checkFuncWithCtx(f *ssa.Function, tp entryType) {
 
 	for _, b := range f.Blocks {
 		for _, instr := range b.Instrs {
+			if !instr.Pos().IsValid() {
```

Comment:
Could we extend tests?

Perhaps the minimum reproducible example is the following:

```go
/* ----------------- issue 21 ----------------- */

func f16(ctx context.Context, k string) func() {
	return func() {
		f16(context.Background(), k)
	}
}
```

We can append it to the file https://github.com/kkHAIKE/contextcheck/blob/master/testdata/src/a/a.go